### PR TITLE
Update react-async-script

### PR DESCRIPTION
**Pull request by nonken**
The referenced version of `react-async-script` was still using React.PropTypes. Updated to latest.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,2 @@
 node_modules
-lib
```
Comment (lemieux) [inline .gitignore:2]: This should stay ignored and the files in the folder that were comitted should not be here.

**Review by lemieux — CHANGES_REQUESTED** [review]
Hey! thanks for the contribution :) 

Just need to bring back the ignore rules and remove the files that were committed and I'll publish a new version.

**Comment by jmulans** [comment]
Hi, can this pull request be merged ? we can't update to React 16 with this old version of react-async-script

**Comment by lemieux** [comment]
Sure, I'll close this PR and redo the changes in a branch.

**Comment by lemieux** [comment]
Fixed on v2.0.0 and available on npm.

**Comment by jmulans** [comment]
thanks ! it works perfectly

**Comment by nonken** [comment]
I completely missed this! Thanks for updating and sorry for being very slow responding!

**Pull request closed without merging**